api/v1beta1/types.go

```diff
@@ -313,6 +315,12 @@ type OscVm struct {
 	ClusterName        string                    `json:"clusterName,omitempty"`
 }
 
+type OscRootDisk struct {
+	RootDiskIops int32  `json:"rootDiskIops,omitempty"`
+	RootDiskSize int32  `json:"rootDiskSize, omitempty"`
+	RootDiskType string `json:"rootDiskType, omitempty"`
```

Comment:
Remove the space between ',' and 'omitempty'

Reply:
done